### ISSUE TITLE
`TURN` with `source` collection does not work properly

### ISSUE THREAD

**Issue by 96LawDawg** [ArnoldSmith86/virtualtabletop#2079]
To duplicate, create a room with multiple seats and set the turn to true on one of them.  Then create a collection of one or more of the other seats where turn is not true.  Use `TURN` with the `source` set to the collection.  The turn indicator will move to the next seat in line, but it does not turn off for the one that was originally selected.  This is probably the same problem from #1954 and #1955.

**Comment by ArnoldSmith86** [comment]
Is this a bug or a feature though? I would have expected `TURN` to not touch any seats outside the `source` collection so that I could manage multiple sets of seats independently.

_But_: I never actually needed that anywhere and you clearly stumbled upon a situation where it makes more sense to change the other seats. For what did you want to use it?

**Comment by 96LawDawg** [comment]
In Searching for Sasquatch, the turn indicator is complicated.  Say it is my turn.  I can take one of two actions.  I can ask a specific player to do something (in which case I want the turn indicator to move to that player until the action is complete and then the turn indicator moves to the next player). Or I can trigger a series of actions that go around the table in order until it gets back around to me.  I want the turn indicator to cycle through each player in order. But in that second scenario, some players may be skipped.  So I wanted to create a collection of those players who still had to make a play and cycle the turn indicator among them. But that left the turn indicator for me still turned on.  I found a workaround.

I thought of this as a bug. I never thought it might be a feature. In fact, I assumed I broke it when I worked on 1955. But I can't think of an easy way to test if your way was the actual behavior before.
